README.md

```diff
@@ -41,14 +41,18 @@ make serve
 
 You can browse the development server at http://localhost:1313, and any changes you make to content or layouts should be reloaded automatically.
 
-## Linting and testing
+## Linting and formatting
 
 To check your code and your Markdown files for issues before submitting, run:
 
 ```
 make lint
 ```
 
+This repo uses Prettier for code formatting, to keep styling + formatting issues aligned without the need to be reviewed in PRs.  We currently do not automatically run the formatter as a build check, but may opt to do so in the future as needed.  To see what files have formatting diffs, you can run `make format` from the project root, which will list all the files with diffs.  To fix these diffs in bulk, you can run `yarn prettier --write .` also from the root.
```

Comment:
FWIW, I would run lint in the build. And _maybe_ have targets or yarn scripts that both _check_ the formatting (returning non-zero if changes are proposed) and _does_ the formatting, but that might be overkill.